epub34/rs/index.html

```diff
@@ -1205,22 +1205,29 @@ <h3>Cascading Style Sheets (CSS)</h3>
 							[=EPUB content documents=].</p>
 					</li>
 					<li>
-						<p id="confreq-css-overrides">MAY override parts of the creator's style sheets (for, e.g., ergonomic or 
-							user interface reasons). If it does, it SHOULD preserve as much as possible the cascade model of CSS.
-							Reading system developers are expected to also publicly document the user agent style sheets and how they
-							interact with EPUB creator's style settings.</p>
-					</li>
-					<li>
-						<p id="confreq-css-user-styles">
-							It MAY provide the user with an affordance that reverts the display to the author's stylesheet.</p>
+						<p id="confreq-css-rs-html-default">SHOULD support the [[html]] <a data-cite="html#rendering"
+								>suggested default rendering</a> in their user agent style sheet(s).</p>
 					</li>
 				</ul>
 
-				<p id="confreq-css-rs-html-default">In addition to supporting CSS properties as defined above, a reading
-					system's user agent style sheet SHOULD support the [[html]] <a data-cite="html#rendering">suggested
-						default rendering</a>.</p>
+				<div class="note">
+					<p>Reading system developers are expected to publicly document their user agent style sheet(s) and
+						how they interact with EPUB creator's style settings. </p>
+				</div>
+
+				<section id="sec-css-overrides">
+					<h4>Author styling overrides</h4>
+
+					<p id="confreq-css-overrides">A reading system MAY override parts of an [=EPUB publication|EPUB
+						publication's] styling (e.g., for ergonomic or user interface reasons), but SHOULD use
+						CSS-compatible methods (user agent style sheet, or [[html]] [^style^] attributes) that allow the
```

Comment:
To question my own pull request... do we really need to say anything about css here? Could this recommendation just be to use methods that allow the author's original design to be rendered? In other words, just don't be destructive in restyling however you choose to do it.